api/sessions/remove.js

```diff
@@ -0,0 +1,20 @@
+module.exports = removeSession
+
+var deleteSession = require('../../utils/session/delete')
+
+function removeSession (state, id, options) {
+  return new Promise(function (resolve, reject) {
+    deleteSession({
+      couchUrl: state.url,
+      bearerToken: id,
+      includeAccount: options.include === 'account',
+      includeProfile: options.include === 'account.profile'
```

Comment:
do we need to mention in [here](http://docs.accountrestapi.apiary.io) that `?include=account` and `?include=account.profile` are supported?